rosa_hcp/terraform/rosa-hcp-creating-a-cluster-with-customizations-terraform.adoc

```diff
@@ -0,0 +1,72 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="rosa-sts-creating-a-cluster-with-customizations-terraform-hcp"]
+= Customizing a ROSA cluster with Terraform
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-sts-creating-a-cluster-with-customizations-terraform-hcp
+
+toc::[]
+
+You can use Terraform to create a customized {product-title} (ROSA) cluster by using Terraform to customize the cluster's details.
+
+[NOTE]
+====
+This guide assumes a basic understanding of ROSA. See the additional resources for conceptual information about ROSA clusters.
+====
+
+.Prerequisites
+
+* You have generated and stored an link:https://console.redhat.com/openshift/token[offline {cluster-manager-first} token].
+* You have an AWS account with the correct permissions.
+* You completed the xref:../../rosa_planning/rosa-cloud-expert-prereq-checklist.adoc[Prerequisites checklist for deploying ROSA using STS]
+* You completed the setup steps in xref:../../rosa_planning/rosa-understanding-terraform.adoc[Preparing Terraform to install ROSA clusters]
+* You have installed link:https://developer.hashicorp.com/terraform/install[the `terraform` CLI tool].
+
+[id="rosa-sts-creating-a-cluster-with-customizations-terraform-creating-hcp"]
+== Creating a ROSA Classic cluster using Terraform
```

Comment:
```suggestion
== Creating a ROSA (classic architecture) cluster using Terraform
```
If this is a module for Classic, all instances of "ROSA" will need to be updated.